js/DropZone/DropZoneFileUtils.js

```diff
@@ -0,0 +1,56 @@
+class DropZoneFileUtils {
+    /**
+     * Format filename for printing
+     * @param  {String} filename
+     * @return {String}
+     */
+    getFileName (filename) {
+        return filename.replace(/.*[\\\/]/, '');
+    }
+
+    /**
+     * Format type for printing
+     * @param  {String} type
+     * @return {String}
+     */
+    getFileType (type) {
+        if (type.length) {
+            return type;
+        } else {
+            return 'application/file';
```

Comment:
This is a "presentational" MIME, so for files we can't detect this was going to be string printed on the front end, I'm thinking maybe this should just return an empty string an not display a type for types we can't match.